Automattic/wp-calypso
# Refactor away from Lodash functionality in bin/audit-svg-colors

## Description

This change is for a utility that audits .svg files. I have modified the code to remove the Lodash functions from this component.

## Proposed Changes

* Replaced the following Lodash functions with the native JS code
1. _.pickBy
2. _.sortBy
3. _.forInRight
4. _.startsWith

Why?
Lodash is known to inflate the bundle size of packages unnecessarily, and in most cases, it can be replaced with native language functionality. 

## Testing Instructions

- Open the file bin/audit-svg-colors.js
- Place the breakpoints at all the lines where the Lodash functions are replaced with the native JS code.
- Click on Run and Debug
- Ensure the values returned by the new functionality are the same as those returned with the Lodash functions.


## Checklist
Verify all tests still pass.
